docs/ManualDeploymentGuideScalarDLOnAKS.md

```diff
@@ -1,74 +1,53 @@
-
-* [Create an AKS cluster for Scalar Products]() // TODO: Create a new document
+ScalarDL Ledger uses ScalarDB in its internal to access a database and ScalarDB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy ScalarDL Ledger. See [Set up a database for ScalarDB/ScalarDL deployment in Azure](./SetupDatabaseForAzure.md) for more details.
 
 ## Step 4. Create a bastion server
 
-For executing some tools to deploy and manage Scalar DL Ledger on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 3**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDL Ledger on AKS, you need to prepare a bastion server in the same VNet of the AKS cluster you created in **Step 2**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 5. Prepare a custom values file of Helm 
 
-You need to configure a custom values file for the Helm Chart of Scalar DL Ledger and Scalar DL Schema Loader based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
-
-## Step 6. Deploy Scalar DL Ledger using Scalar Helm Chart
-
-Deploy Scalar DL Ledger on your AKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+You need to configure a custom values file for the Helm Chart of ScalarDL Ledger and ScalarDL Schema Loader based on your environment (e.g., access information of the database you created in **Step 3**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
 
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 6. Deploy ScalarDL Ledger using Scalar Helm Chart
 
-## Step 7. Check the status of Scalar DL Ledger deployment
+Deploy ScalarDL Ledger on your AKS cluster using Scalar Helm Chart. See [Deploy Scalar products using Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-After deploying Scalar DL Ledger on your AKS cluster, you need to check the status of each component. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardl-ledger` command and deploying ScalarDL Ledger in the namespace by using the `-n scalardl-ledger` option with the `helm install` command.
```

Comment:
I added a new recommendation based on the offline discussion. We recommend using the dedicated k8s namespace for Scalar product deployment.